.gitlab-ci.yml

```diff
@@ -970,16 +970,34 @@ notify:publish-release-failure:
     - 'MESSAGE_TEXT=":status_alert: $CI_PROJECT_NAME $CI_COMMIT_TAG publish pipeline <$BUILD_URL|$COMMIT_MESSAGE> failed."'
     - postmessage "#mobile-sdk-ops" "$MESSAGE_TEXT"
 
+notify:prepare-github-token:
+  tags: [ "arch:amd64" ]
+  only:
+    - tags
+  image: $CI_IMAGE_DOCKER
+  stage: notify
+  when: on_success
+  script:
+    - aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.gh_app_private_key --with-decryption --query "Parameter.Value" --out text >> ./gh_private_key.pem
```

Comment:
is it safer to pipe it to the `create_github_installation_token.sh` instead of creating a file? or it doesn't matter since the runner is destroyed after?